crates/net/network/src/manager.rs

```diff
@@ -632,301 +890,84 @@ where
     type Output = ();
 
     fn poll(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
+        let start = Instant::now();
+        let mut poll_durations = NetworkManagerPollDurations::default();
+
         let this = self.get_mut();
 
-        // poll new block imports
+        // poll new block imports (dummy)
```

Comment:
we don't know this because the type is boxed, we can however add a note that this is expected to be a noop for POS
```suggestion
        // poll new block imports
```

Reply:
will add that note